src/mono/wasm/runtime/managed-exports.ts

```diff
@@ -188,17 +188,8 @@ export function init_managed_exports(): void {
             Module.stackRestore(sp);
         }
     };
-
-    if (MonoWasmThreads && install_sync_context) {
-        runtimeHelpers.javaScriptExports.install_synchronization_context = () => {
-            const sp = Module.stackSave();
-            try {
-                const args = alloc_stack_frame(2);
-                invoke_method_and_handle_exception(install_sync_context, args);
-            } finally {
-                Module.stackRestore(sp);
-            }
-        };
+    if (MonoWasmThreads && install_main_synchronization_context) {
```

Comment:
do we need the && here since we already assert that it's defined further up? not a big deal though

Reply:
Think this may help tree shaking and also I like the clarity of it.